src/designsystem/Button.tsx

```diff
@@ -1,52 +1,38 @@
 import React from 'react';
-import './button.css';
 
-interface ButtonProps {
-  /**
-   * Is this the principal call to action on the page?
-   */
-  primary?: boolean;
-  /**
-   * What background color to use
-   */
-  backgroundColor?: string;
-  /**
-   * How large should the button be?
-   */
-  size?: 'small' | 'medium' | 'large';
-  /**
-   * Button contents
-   */
-  label: string;
-  /**
-   * Optional click handler
-   */
-  onClick?: () => void;
+interface ButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
+  isDisabled?: boolean;
+  color?: 'primary' | 'secondary';
+  size?: 'medium' | 'large';
+  children?: React.ReactNode;
 }
 
-/**
- * Primary UI component for user interaction
- */
 export const Button = ({
-  primary = false,
-  size = 'medium',
-  backgroundColor,
-  label,
-  ...props
+  isDisabled = false,
+  color = 'primary',
+  size = 'large',
+  children,
+  ...rest
 }: ButtonProps) => {
-  const mode = primary ? 'storybook-button--primary' : 'storybook-button--secondary';
   return (
     <button
       type="button"
-      className={['storybook-button', `storybook-button--${size}`, mode].join(' ')}
-      {...props}
+      disabled={isDisabled}
+      className={`w-full font-bold rounded-full 
+        ${isDisabled ? 'opacity-50' 
+        : (!isDisabled && color === 'primary') ? 'hover:bg-primary-btn-hover' 
+        : (!isDisabled && color === 'secondary') ? 'hover:bg-secondary-btn-hover' 
+        : null}
+        ${color === 'primary' ? 'bg-primary-btn text-white' 
+        : color === 'secondary' ? 'bg-secondary-btn text-secondary-btn-label' 
+        : null}
+        ${size === 'medium' ? 'py-2.5 px-4 text-base leading-5' 
+        : size === 'large' ? 'py-4 px-7 text-lg leading-6' 
+        : null}
+        `}
```

Comment:
question: would it be better if we use `https://github.com/lukeed/clsx#readme` here?